plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -344,8 +351,8 @@ function setupInternalRoutes(
     },
   );
 
-  router.get('/instances', async (_, res) => {
-    await V1.getInstances(services.dataIndexService)
+  router.get('/instances', async (req, res) => {
+    await V1.getInstances(services.dataIndexService, buildPagination(req))
```

Comment:
please remove the changes from V1

Reply:
[743cfca](https://github.com/janus-idp/backstage-plugins/pull/1313/commits/743cfcaa764696587fc59150f8291cc1d1387123)